src/Web.Application/dockerfile.prd

```diff
@@ -2,5 +2,6 @@ FROM microsoft/dotnet:1.1-runtime
 WORKDIR /app
 COPY . /app
 ENV ASPNETCORE_URLS http://*:5000
```

Comment:
I think, we can use special docker image https://hub.docker.com/r/microsoft/aspnetcore/ rather than common  microsoft/dotnet

Reply:
@SStolbov, have a look, please